tests/bluetooth/bsim_bt/compile.source

```diff
@@ -1,6 +1,12 @@
 # Copyright 2018 Oticon A/S
 # SPDX-License-Identifier: Apache-2.0
 
+function print_error_info(){
+  echo -e "\033[0;31mFailure building ${app} ${conf_file} for ${BOARD}\033[0m\n\
+  You can rebuild it with\n\
+  ${cmake_cmd[@]} && ninja ${ninja_args}"
```

Comment:
Isn't `west` the default build method in Zephyr? Shouldn't we recommend that?

Reply:
That would be another change :), this just prints what the script does today

Reply:
True, but given that is what the Zephyr documentation suggests, I think we should in the future update this to use `west` instead :)